File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        gt = {}
        for i in range(len(nums)):
            if nums[i] in gt:
                return [gt[nums[i]], i]
            else:
                gt[target - nums[i]] = i

File: Data Structures & Algorithms/duplicate-integer/submission-0.cs
public class Solution {
    public bool hasDuplicate(int[] nums) {
        Dictionary<int, int> count = new Dictionary<int, int>();

        foreach (int num in nums) {
            if (count.ContainsKey(num)) {
                return true;
            }
            count[num] = 1;
        }

        return false;
    }
}